source/_patterns/02-organisms/components/hero-banner.mustache

```diff
@@ -0,0 +1,54 @@
+<div class="hero-banner" role="region" data-behaviour="Hero-banner" {{#headingId}} aria-labelledby="{{headingId}}" {{/headingId}}>
```

Comment:
@discodavey can the `aria-labelledby` have a fixed value? Can you provide an example of what this might be set to? Is `headingId` the best property name for this?

Reply:
Role should be banner

Reply:
Using a aria-labelledby is referencing another element connected with it. ie. and ID. Is the Hero referenced by other element? If not then it can be removed

Reply:
Role attribute is changed to banner. Hero banner is not referenced by other element so I remove the aria-labelledby attribute.